game/scripts/npc/items/item_allied_cyclone_2.txt

```diff
@@ -88,5 +88,63 @@
         "cast_range_tooltip"                              "575"
       }
     }
+      "OnSpellStart"
+    {
+
+      "ApplyModifier"
+      {
+        "Target"                                          "TARGET"
+        "ModifierName"                                    "modifier_item_allied_cyclone_2_active"
+        "Duration"                                        "%allied_cyclone_duration"
+      }
+    }
+    "Modifiers"
+    {
+      "modifier_item_allied_cyclone_2"
+      {
+        "Passive"                                         "1"
+        "IsHidden"                                        "1"
+        "Properties"
+        {
+          "MODIFIER_PROPERTY_STATS_INTELLECT_BONUS"         "%bonus_intellect"
+          "MODIFIER_PROPERTY_MANA_REGEN_PERCENTAGE"         "%bonus_mana_regen"
+          "MODIFIER_PROPERTY_MOVESPEED_BONUS_CONSTANT"      "%bonus_movement_speed"
+        }
+      }
+      "modifier_item_allied_cyclone_2_active"
```

Comment:
I don't think there's any reason to name the different level modifiers differently.